runner/start.sh

```diff
@@ -1,12 +1,26 @@
-DIR=jars
-if [ ! -d "$DIR" ]; then
-    echo "$DIR does not exists, creating..."
-    mkdir jars
-fi
-FILE=./jars/core.jar
-if [ ! -f "$FILE" ]; then
-    echo "$FILE does not exists, creating..."
-    ./update.sh
-fi
+set -exu
 
-cd jars && java -jar core.jar "$@"
+PROJECT_ROOT=${PROJECT_ROOT:-$(git rev-parse --show-toplevel)}
+
+pushd "$PROJECT_ROOT"
+  pushd runner
+    DIR=jars
+    if [ ! -d "$DIR" ]; then
+        echo "$DIR does not exists, creating..."
+        mkdir jars
+    fi
+    FILE=./jars/core.jar
+    if [ ! -f "$FILE" ]; then
+        echo "$FILE does not exists, creating..."
+        ./update.sh
+    fi
+  popd
+popd
+
+pushd "$PROJECT_ROOT"
+  pushd runner
+    pushd "$DIR"
+      java -jar core.jar "$@"
+    popd
+  popd
+popd
```

Comment:
Why isn't this ending in a newline?

Reply:
8f79e985a4d0f416d63cd465a247ac3ffe11a573